go.mod

```diff
@@ -366,4 +442,10 @@ require (
 )
 
 replace github.com/libgit2/git2go/v33 => ./git2go
-replace google.golang.org/grpc => google.golang.org/grpc v1.54.0
\ No newline at end of file
+
+replace (
+	// TODO(vladklokun): Since later versions (e.g. v0.40.0) that get used without the pin introduce weird packaging issues probably due to package renames, pin to last known good.
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.35.0
+
+	google.golang.org/grpc => google.golang.org/grpc v1.54.0
+)
```

Comment:
Please take note that it seems that something in OpenTelemetry broke their package structure, so I had to pin the versions.